src/azul/drs.py

```diff
@@ -147,8 +147,9 @@ def _get_object_access(self,
             response = self._request(url)
             if response.status == 200:
                 response = json.loads(response.data)
-                return Access(method=access_method,
```

Comment:
Even if we can assume that a URL returned by GET /objects/{object_id}/access/{access_id} is always `https`—which I don't think we can, the standard says nothing about that—we should encode that here, not as a default in the Access class.